build.gradle

```diff
@@ -11,8 +11,11 @@ sourceCompatibility = JavaVersion.VERSION_1_6
 dependencies {
   compile 'io.dropwizard.metrics:metrics-core:3.1.0'
   compile 'com.newrelic.agent.java:newrelic-api:3.10.0'
-
+  compile 'com.google.guava:guava:18.0'
```

Comment:
Oh, I just meant a subproject in the gradle sense: create the normal multi-project gradle structure, not a separate repo and "top level" project. I can do that if you're not comfortable with the process.